cowprotocol/accounting/cost_coverage/cost_coverage_2601653.sql

```diff
@@ -0,0 +1,117 @@
+    group by evt_block_time
+),
+
+fees_eth as (
+    select
+        max(block_time) as latest_withdrawal_time,
+        sum(fees) as fees
+    from (
+        select
+            block_time,
+            fees
+        from purchased_eth
+        union all
+        select
+            block_time,
+            fees
+        from transferred_eth
+    )
+),
+
+outgoing_eth as (
+    select sum(value / 1e18) as eth
+    from ethereum.traces
+    where
+        "from" = 0xa03be496e67ec29bc62f01a428683d7f9c204930
+        and success = true
+        and (lower(call_type) not in ('delegatecall', 'callcode', 'staticcall') or call_type is null)
+        and block_time between (timestamp '{{start_time}}' + interval '2' day) and (timestamp '{{end_time}}' + interval '2' day)
+        -- Excluding 80 ETH transfer due to 
+        -- https://snapshot.org/#/cow.eth/proposal/0x79fdcc006030d50ab0ffe0ffd7c474a409eb70448d1c8eba58919af7559a876e
+        -- https://etherscan.io/tx/0x86f101b7ffa11c734ffe117bb2f0e4b377260f3fc3d8517e91d01126eed12980
+        and tx_hash != 0x86f101b7ffa11c734ffe117bb2f0e4b377260f3fc3d8517e91d01126eed12980
+),
+
+outgoing_cow as (
+    select sum(value) / 1e18 as cow
+    from erc20_ethereum.evt_Transfer
+    where
+        contract_address = 0xdef1ca1fb7fbcdc777520aa7f396b4e015f497ab
+        and "from" = 0xa03be496e67ec29bc62f01a428683d7f9c204930
+        and evt_block_time between (timestamp '{{start_time}}' + interval '2' day) and (timestamp '{{end_time}}' + interval '2' day) and evt_block_number != 19182562
+),
+
+conversion_prices as (
+    select
+        (
+            select avg(price) from prices.usd
+            where
+                blockchain = 'ethereum'
+                and contract_address = 0xdef1ca1fb7fbcdc777520aa7f396b4e015f497ab
+                and date(minute) = cast('{{end_time}}' as timestamp) - interval '1' day
+        ) as cow_price,
+        (
+            select avg(price) from prices.usd
+            where
+                blockchain = 'ethereum'
+                and contract_address = 0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2
+                and date(minute) = cast('{{end_time}}' as timestamp) - interval '1' day
+        ) as eth_price
+),
+
+protocol_fees_collected as (
+    select sum(cast(cast(data.protocol_fee as varchar) as int256) * data.protocol_fee_native_price / pow(10, 18) - coalesce(case when data.partner_fee_recipient is not null then cast(data.partner_fee as int256) * data.protocol_fee_native_price / pow(10, 18) end, 0)) as protocol_fee_in_eth --noqa: RF01
+    from cowswap.raw_order_rewards as r
+    inner join ethereum.blocks as b on number = block_number
+    where
+        r.block_number > 19068880
+        and data.protocol_fee_native_price > 0 --noqa: RF01
+        and b.time between timestamp '{{start_time}}' and timestamp '{{end_time}}'
+        and r.order_uid not in (select cast(order_uid as varchar) from query_3639473)
+),
```

Comment:
This table does not seem to be included in the final result. We should either add it there or remove this code.

Reply:
Good catch. Removing it

Reply:
I believe it does. It is used in the `results` table, and then we `select *` from `results` at the very end.